cookie_consent/templatetags/cookie_consent_tags.py

```diff
@@ -124,6 +121,7 @@ def js_type_for_cookie_consent(request, varname, cookie=None):
         alert("Social cookie accepted");
       </script>
     """
+    # FIXME: clean up the script types?
```

Comment:
Still needed for the legacy support, but in the next next minor release we should be able to get rid of it?

Reply:
I've deprecated it and added playwright tests to ensure it doesn't break because of lack of tests